### PR TITLE
Add AFI 100 ranking

### PR DESCRIPTION
**Pull request by emmanuelgautier1994**

### CODE REVIEW

File: lib/afi.rb [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    doc = Nokogiri.HTML(html)
+    movies_list = doc.css('.q_title')[0...100]
+
+    movies_list.each do |movie|
+      ranking, _, title, _, year = movie.content.match(/(\d+)(\.\s+)([^\(]+\S)(\s+\()(\d{4})(\)$)/).captures
+  
+      tmdb_movie = Tmdb.search_movie(title, year)
+      movie_record = Movie.find(imdb_id: tmdb_movie['imdb_id'])
+
+      puts title + ' is ranked #' + ranking + ' on AFI 100 list'
```
Comment (coisnepe) [inline lib/afi.rb:20]: What about interpolation?

```suggestion
      puts "#{title} is ranked ##{ranking} on AFI 100 list"
```

File: lib/afi.rb [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+      else
+        puts '++++ Creating new record for ' + title
+        Movie.insert(imdb_id: tmdb_movie['imdb_id'], afi_100_ranking: ranking.to_i)
+      end
+
+      rescue MovieNotFound
+        Movie.insert(
+          title: title,
+          year: year.to_i,
+          senscritique_ranking: ranking.to_i
```
Comment (coisnepe) [inline lib/afi.rb:34]: 👀

File: lib/afi.rb [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+    movies_list.each do |movie|
+      ranking, _, title, _, year = movie.content.match(/(\d+)(\.\s+)([^\(]+\S)(\s+\()(\d{4})(\)$)/).captures
+  
+      tmdb_movie = Tmdb.search_movie(title, year)
+      movie_record = Movie.find(imdb_id: tmdb_movie['imdb_id'])
+
+      puts title + ' is ranked #' + ranking + ' on AFI 100 list'
+      
+      if movie_record
+        puts '==== Found existing record for ' + title
```
Comment (coisnepe) [inline lib/afi.rb:23]:
```suggestion
        puts "==== Found existing record for #{title}"
```

File: lib/afi.rb [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+        movie_record.update(afi_100_ranking: ranking.to_i)
+      else
+        puts '++++ Creating new record for ' + title
```
Comment (coisnepe) [inline lib/afi.rb:26]:
```suggestion
        puts "++++ Creating new record for #{title}"
```

File: lib/afi.rb (hunk 2 above)
Comment (emmanuelgautier1994) [reply lib/afi.rb:34]: U_U

File: lib/afi.rb (hunk 1 above)
Comment (emmanuelgautier1994) [reply lib/afi.rb:20]: Don't know what I was thinking ^^"

**Pull request merged**